### PR TITLE
Task/update readme with release info

### PR DESCRIPTION
**Pull request by lbk003**
Added instructions for releasing to PyPi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,86 @@
...
+print(shabad)
+
+```
+Search shabad by first letters"
+```
+print("\tSearching Bandhana Har Bandhana ....")
+shabad = banidb.search("bhbgggr")
+print(shabad)
+```
+
```
Comment (MySmilingTurban) [inline README.md:31]: Shall we add all the available functions and uses to make it more user-friendly?

Comment (lbk003) [reply README.md:31]: Yes, we should. Would you like to create an issue to track that, and then add it?

Comment (lbk003) [reply README.md:31]: I can merge this one as is, if you like this approach?

Comment (MySmilingTurban) [reply README.md:31]: Ya, that would be great 👍🏽

**Review by MySmilingTurban — APPROVED** [review]

**Pull request merged**